src/App.vue

```diff
@@ -144,7 +150,54 @@ export default {
       stelace.onError('userSessionExpired', () => {
         debouncedEmitUserSessionExpiredError()
       })
-    }
+    },
+    // messages received from Stelace Dashboard
+    receiveMessage (event) {
+      const origins = this.allowedMessageOrigins.split(',')
```

Comment:
What about potential white spaces? You probably need `.map(o => o.trim())`.

Reply:
Ok for handling white spaces.